en/document-v1-api-guide.html

```diff
@@ -145,7 +145,7 @@ <h2 id="data-dump">Data dump</h2>
     "continuation": "AAAAEAAAAAAAAAM3AAAAAAAAAzYAAAAAAAEAAAAAAAFAAAAAAABswAAAAAAAAAAA"
 }
 </pre>
-Note the <em>continuation</em> token - use this in the next request for more data.
+Note the <em>continuation</em> token — use this in the next request for more data.
```

Comment:
I see the "Elements of typographic style" – a most excellent book on the subject matter – recommends not using em dashes at all; they're simply too long, and belong rather with the "corseted aesthetics of the Victorian age".

Reply:
Em dashes do exude a more \~🦄 _classical_ 🦄\~ vibe, so in general I suppose I'd more expect to see them in serif'd prose than in sans serif'd technical writing. As such I'm fine with whatever.

As a meta-comment, _Practical Typography_ (a very good read—incidentally also saying it's OK to use spaces around em dashes) says;

> Fans of Robert Bringhurst’s book _The Elements of Typographic Style_ (and I am among them—see bibliography) may know that he recommends using en dashes with spaces rather than em dashes. Perhaps this practice is common in Mr. Bringhurst’s native Canada. But in American typography, it’s not.
>
> (https://practicaltypography.com/hyphens-and-dashes.html)

So no hard rules.

Reply:
I also find here:

> Em and en dashes are typically set flush against the surrounding text. Some fonts include a little white space around the em dash; some don’t. If your em dashes look like they’re being crushed—particularly if you’re setting type on screen—it’s fine to add word spaces before and after.

Perhaps that's my problem — the font used, e.g., here, makes it seem crushed.

Reply:
I mean, it's just seconds until they all go ~

Reply:
(Certainly your reference, above ... 🤦)